backend/services/calendar_service.py

```
@@ -41,5 +72,7 @@ async def create_calendar_event(todo_text: str, user_token: dict) -> dict:
         return created_event
     except UnsafeCalendarTodoError:
         raise
+    except CalendarServiceError:
+        raise
     except Exception as e:
         raise CalendarServiceError(f"Failed to create event: {str(e)}") from e
```

Ruff (0.15.14)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)